doc/releases/changelog-dev.md

```diff
@@ -868,6 +868,8 @@
 
 * To standardize across all optimizers, `qml.optimize.AdamOptimizer` now also uses `accumulation` (in form of `collections.namedtuple`) to keep track of running quantities. Before it used three variables `fm`, `sm` and `t`. [(#1757)](https://github.com/PennyLaneAI/pennylane/pull/1757)
 
+* AngleEmbedding now supports `batch_params` decorator. [(#1812)](https://github.com/PennyLaneAI/pennylane/pull/1812)
+
```

Comment:
Hi @KetpuntoG, this addition now will be a part of the v0.20.0 release and the `doc/releases/changelog-dev.md` file has been reset. Could you re-add this entry by merging the `master` branch into this feature branch? :slightly_smiling_face: